loadbalancer/etcd/publisher_test.go

```diff
@@ -0,0 +1,107 @@
+package etcd
```

Comment:
This will let the tests fail until its its merged in go-kit master, cause of the import statement `kitetcd "github.com/go-kit/kit/loadbalancer/etcd" or do i have you note wrong about the "disambiguate"?